src/uct/ib/mlx5/gdaki/gdaki.h

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2025. ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCT_GDAKI_IFACE_H
+#define UCT_GDAKI_IFACE_H
+
+#include <uct/base/uct_iface.h>
+#include <uct/cuda/base/cuda_iface.h>
+#include <uct/ib/base/ib_iface.h>
+#include <uct/ib/mlx5/ib_mlx5.h>
+#include <uct/ib/rc/base/rc_iface.h>
+
+#include <doca_gpunetio.h>
+
+typedef struct uct_gdaki_iface {
+    struct doca_gpu *gpu_dev;
+    CUdevice cuda_dev;
```

Comment:
align
should it inherit from ib_iface?

Reply:
on uct_rc_mlx5_iface_common_t or uct_rc_mlx5_iface_t probably?
I planned to add it next PR.